README.md

```diff
@@ -1,3 +1,4 @@
+[![Build Status](https://travis-ci.org/googlei18n/libphonenumber.svg?branch=master)](https://travis-ci.org/googlei18n/libphonenumber)
```

Comment:
Can you make it right-aligned? It's very nice as a status for us, but it's not what most clients want to see first.